jstreegrid.js

```diff
@@ -216,7 +218,7 @@
 					gs.gridcontextmenu = false;
 				}
 				// find which column our tree shuld go in
-				for (i=0;i<s.columns.length;i++) {
+        for (var i = 0, len = s.columns.length;i<len;i++) {
```

Comment:
I expect almost nothing here. A good engine should optimize this away (although not sure, since it would need to evaluate each time... whatever, leave it).